tableau-databricks/connection-builder.js

```diff
@@ -29,15 +29,24 @@ limitations under the License.
 	var authenticationMode = attr[connectionHelper.attributeAuthentication];
 	switch (authenticationMode) {
 		case "auth-user-pass":
+		case "username-password":
 			params["AUTHMECH"] = 3;
 			params["UID"] = attr[connectionHelper.attributeUsername];
 			params["PWD"] = attr[connectionHelper.attributePassword];
 			break;
+
 		case "auth-pass":
 			params["AUTHMECH"] = 3;
 			params["UID"] = "token";
 			params["PWD"] = attr[connectionHelper.attributePassword];
 			break;
+
+		case "oauth":
+			params["AUTHMECH"] = 11;
+			params["AUTH_FLOW"] = 0; // token passthrough
+			params["AUTH_ACCESSTOKEN"] = attr["ACCESSTOKEN"];
+			break;
+
```

Comment:
I think `spark.sql.crossJoin.enabled` is enabled by default starting with 7.0. Perhaps we could remove it ...